modules/ROOT/pages/kernel-args.adoc

```diff
@@ -1,10 +1,87 @@
 = Modifying Kernel Arguments
 
-Currently to change kernel arguments, you must script a systemd service which runs `rpm-ostree kargs --reboot`. The command supports appending kernel arguments (via `--append KEY[=VAL]`), deleting them (via `--delete KEY[=VAL]`), and replacing them (via `--replace KEY[=VALUE]`).
+Kernel arguments changes are managed by `rpm-ostree` via the https://www.mankier.com/1/rpm-ostree[`rpm-ostree kargs`] sub command. Changes are applied to a new deployment and a reboot is necessary for those to take effect.
```

Comment:
Aside: feels funny to link to a third-party rendering of our own man pages. Maybe we should figure out a way to render them in https://coreos.github.io/rpm-ostree/.

Reply:
Yep!